client/account-recovery/index.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import controller from './controller';
+
+export default function() {
```

Comment:
Might as well name this :D, it'll help if we export any other functions in this file